scania-digital-design-system/sdds
# AB#476  Added textarea styling

- Extends styling from textfield

**How to test**
- Check design in Figma
- Check if textarea functions properly


**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: [AB#476](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/476)
